.github/workflows/upload-tutorials-stats.yml

```
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Page age
+
+# Controls when the workflow will run
+on:
+  schedule:
+  # Run this once per day.
+    - cron: "0 0 * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  get-tutorials-stats:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+        repository: 'pytorch/tutorials'
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+      - name: Install dependancies
+        run: |
+          python3 -m pip install boto3 typing
```

Pinning would look like:

```suggestion
          python3 -m pip install boto3==1.26.69 typing==3.10.0.0
```